### PR TITLE
qt: Move BitcoinGUI initializers to class, fix initializer order warning

### PR DESCRIPTION
**Pull request by laanwj**
- C++11-ize the code (move initializers to class, change `0` to `nullptr` where appropriate)
- Make sure `m_wallet_selector` is initialized
- And fix the following warning:

    bitcoin/src/qt/bitcoingui.cpp:122:5: warning: field 'spinnerFrame' will be initialized after field 'm_wallet_selector_label' [-Wreorder]
        spinnerFrame(0),

### CODE REVIEW

**Comment by promag** [comment]
Heh this was on my TODO list 😆 

utACK 5f0303c.

File: src/qt/bitcoingui.h [hunk 1]
```diff
@@ -87,58 +87,58 @@ class BitcoinGUI : public QMainWindow
...
+    QAction *quitAction = nullptr;
+    QAction *sendCoinsAction = nullptr;
+    QAction *sendCoinsMenuAction = nullptr;
+    QAction *usedSendingAddressesAction = nullptr;
+    QAction *usedReceivingAddressesAction = nullptr;
+    QAction *signMessageAction = nullptr;
+    QAction *verifyMessageAction = nullptr;
+    QAction *aboutAction = nullptr;
+    QAction *receiveCoinsAction = nullptr;
     QAction *receiveCoinsMenuAction;
```
Comment (MarcoFalke) [inline src/qt/bitcoingui.h]: Why is this no longer initialized?

Comment (laanwj) [reply src/qt/bitcoingui.h]: Ouch, I think I forgot that one.

**Comment by laanwj** [comment]
Ok, made sure that `receiveCoinsMenuAction` is initialized.

**Comment by MarcoFalke** [comment]
re-utACK bb3de15ad8deba7b8b9740d9737dfe5bccf4e0e4 Only change is to address the feedback.

**Pull request merged**